python/cudf/cudf/core/column/column.py

```diff
@@ -1613,6 +1620,18 @@ def as_column(arbitrary, nan_as_null=None, dtype=None, length=None):
         data = as_column(
             np.asarray(arbitrary), dtype=dtype, nan_as_null=nan_as_null
         )
+    elif isinstance(arbitrary, pd.core.arrays.masked.BaseMaskedArray):
+        cudf_dtype = arbitrary._data.dtype
+
+        data = Buffer(arbitrary._data.view("|u1"))
+        data = as_column(data, dtype=cudf_dtype)
+
+        mask = arbitrary._mask
+        mask = bools_to_mask(
+            as_column(mask).binary_operator("eq", np.bool_(False))
+        )
```

Comment:
Do we have a `~` unary operator? If so we should be able to use that more efficiently to avoid the jitting of the vector / scalar operation.

Reply:
Looks like we have it on the series level but not the column level. Should we promote to a series and use it then?

Reply:
How much effort would it be to move it down to the column level? That's probably the right answer.

Reply:
Looking into this now.

Reply:
We can just do `unary_operator("not")`, which seems obvious in retrospect.